codecov.yml

```diff
@@ -0,0 +1,6 @@
+ignore:
```

Comment:
Can you also please add threshold 1% so that we do not get all the false negatives on unrelated changes.
See https://docs.codecov.io/docs/commit-status